derive-new-test/Cargo.lock

```diff
@@ -1,38 +0,0 @@
-[root]
```

Comment:
Sure, I could check that back in if you'd like. What is the general rule for updating the stored Cargo.lock, every version bump?

When I initially tried to build the derive-new-test crate, it complained about the dependencies mismatch (the version was bumped and lock files not re-checked-in?), so I had to run cargo update manually.

I guess I haven't seen many rust repos that had the Cargo.lock files checked in so far -- this and the missing gitignore led me thinking that you might have committed them by accident.